### ISSUE TITLE
.Net Standard Increase Request

### ISSUE THREAD

**Issue by bsheairs928** [step-up-labs/firebase-database-dotnet#59]
We are attempting to use this in our solution, but are having issues because we are using .Net 4.5.1, which is .Net Standard 1.2.  I have downloaded the source for this and noticed that the only thing that is in .Net 4.5 is the Unit Test Project.

Because I  am unfamiliar with this type of solution, I am not sure what is needed to increase the .Net Standard to 1.2.

Would it be possible for you to update this to .Net 1.2 Standard? 
Thank you

**Comment by bezysoftware** [comment]
The lower the .net standard version, the more frameworks you can target, so increasing the version doesn't make any sense. 
I just tried installing the package into 4.5.1 and it worked (VS2017).

**Comment by bsheairs928** [comment]
We don't have the issue in Visual Studio but our Automated Compiler started failing due to the following warning:
[ResolveAssemblyReference] C:\TeamCity\buildAgent\work\4bfe9b9a7ec05422\<Project A>.metaproj warning MSB3268: The primary reference "C:\TeamCity\buildAgent\work\4bfe9b9a7ec05422\<Project B>\bin\Release\<Project B>.dll" could not be resolved because it has an indirect dependency on the framework assembly "System.Runtime, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" which could not be resolved in the currently targeted framework. ".NETFramework,Version=v4.5.1". To resolve this problem, either remove the reference "C:\TeamCity\buildAgent\work\4bfe9b9a7ec05422\<Project B>\bin\Release\<Project B>.dll" or retarget your application to a framework version which contains "System.Runtime, Version=4.0.10.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a".

When this started happening the only major change was adding a reference to this dll.  I have checked all other requirements for this project, at least the only listed here, and I was able to narrow it down a little.  

Also the change from .Net Standard 1.1 to 1.2 only removes support for Window 8.0.

Thank you
